src/docs/modules/appendix/pages/release-notes.adoc

```diff
@@ -17,13 +17,29 @@ SPDX-License-Identifier: MPL-2.0
 [[bamm-2.0.0]]
 === Release 2.0.0
```

Comment:
Would it make sense to also include the documentation updates in the release notes or is this just a bonus that we do not mention?

Reply:
Would it also make sense to mention that with the BAMM 2.0.0 the SHACL shapes and validation have been further improved?

Reply:
Ah, specification is mentioned. Would it make sense to mention it in the beginning (it is not the most significant one, but it is one that is easy to understand and skip to the next point)